### ISSUE TITLE
Some hodographs fail to plot 

### ISSUE THREAD

**Issue by lcarlaw** [tjturnage/cloud-radar-server#45]
Occasional failures of `hodo_plot.py` to generate hodographs have been noted, particularly during longer simulations with more radar files due to timeouts during the `GET` request from the synoptic API.  In particular, the error appears during execution of this line `req = requests.get(api_request_url, params=api_args, timeout=10)` within the `mesowest_get_sfcwind` function.  An example error stack trace is below.  

Unsure if this is because up to 4 simultaneous requests are being made per session due to API request limits.  A `try` and `except` block to catch a ReadTimeout exception would keep the scripts from failing.  

```
Traceback (most recent call last):
  File "/opt/miniforge3/envs/cloud-radar/lib/python3.12/multiprocessing/pool.py", line 125, in worker
    result = (True, func(*args, **kwds))
                    ^^^^^^^^^^^^^^^^^^^
  File "/opt/miniforge3/envs/cloud-radar/lib/python3.12/multiprocessing/pool.py", line 48, in mapstar
    return list(map(*args))
           ^^^^^^^^^^^^^^^^
  File "/data/scripts/lcarlaw/cloud-radar-server/scripts/hodo_plot.py", line 227, in create_hodos
    wnspd, wndir = mesowest_get_sfcwind(api_args)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/scripts/lcarlaw/cloud-radar-server/scripts/hodo_plot.py", line 108, in mesowest_get_sfcwind
    req = requests.get(api_request_url, params=api_args, timeout=10)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/miniforge3/envs/cloud-radar/lib/python3.12/site-packages/requests/api.py", line 73, in get
    return request("get", url, params=params, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/miniforge3/envs/cloud-radar/lib/python3.12/site-packages/requests/api.py", line 59, in request
    return session.request(method=method, url=url, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/miniforge3/envs/cloud-radar/lib/python3.12/site-packages/requests/sessions.py", line 589, in request
    resp = self.send(prep, **send_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/miniforge3/envs/cloud-radar/lib/python3.12/site-packages/requests/sessions.py", line 703, in send
    r = adapter.send(request, **kwargs)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/miniforge3/envs/cloud-radar/lib/python3.12/site-packages/requests/adapters.py", line 532, in send
    raise ReadTimeout(e, request=request)
requests.exceptions.ReadTimeout: HTTPSConnectionPool(host='api.synopticdata.com', port=443): Read timed out. (read timeout=10)
```

**Comment by scottthomaswx** [comment]
Thanks for brining this to my attention, I'll take a look on my mid tonight

**Comment by scottthomaswx** [comment]
Addressed by Pull Request #47
